pkg/kritis/review/review.go

```diff
@@ -146,7 +146,7 @@ func (r Reviewer) findUnsatisfiedAuths(image string, auths []v1beta1.Attestation
 	notAttestedBy := []v1beta1.AttestationAuthority{}
 	for _, auth := range auths {
 		transport := AttestorValidatingTransport{Client: r.client, Attestor: auth}
-		attestations, err := transport.GetValidatedAttestations(image, auths)
+		attestations, err := transport.GetValidatedAttestations(image, &auth)
```

Comment:
'transport' is already initialized with 'auth', so no need to pass it in here as well.  Please see comment below in validated_transport.go.

Reply:
This has now been addressed